src/test/application/BorosTcfTest.js

```diff
@@ -0,0 +1,65 @@
+import {BorosTcf} from '../../main/application/BorosTcf.js'
+import sinon from 'sinon'
+import {expect} from 'chai'
+
+describe('BorosTcf should', () => {
+  const getVendorListUseCase = {
+    execute: () => null
+  }
+  const getConsentStatusUseCase = {
+    execute: () => null
+  }
+  const loadUserConsentUseCase = {
+    execute: () => null
+  }
+  const saveUserConsentUseCase = {
+    execute: () => null
+  }
+
+  const borosTcf = new BorosTcf({
+    getVendorListUseCase,
+    getConsentStatusUseCase,
+    loadUserConsentUseCase,
+    saveUserConsentUseCase
+  })
+
+  it('getVendorList should delegate to getVendorListUseCase', () => {
+    const getVendorListUseCaseSpy = sinon.spy(getVendorListUseCase, 'execute')
```

Comment:
I would have created all spys before the its of the tests, and I would have injected the spys to BorosTcf. In my opinion it would be more understandable.

Reply:
I have tested to inject the spies and don't work. 
![Screenshot 2020-05-07 at 14 44 31](https://user-images.githubusercontent.com/537656/81295856-59c4df00-9071-11ea-9746-fd9397b5d20a.png)

Reply:
ok! sorry